### ISSUE TITLE
[Accessibility]: No notification of error or success after adding or removing code host credentials

### ISSUE THREAD

**Issue by courier-new** [sourcegraph/sourcegraph#34484]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34073

### Problem description

When I am adding my PAT for a code host and I click "Add credential" from the modal, it is not clear whether the operation succeeds or fails.

### Expected behavior

If the operation fails, the error should be focused and dictated by the screen reader.

If the operation succeeds, there should be some sort of visual alert and indication of a status change. We could make use of the [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) attribute to support this on the screen reader.

On successfully adding credentials, the screen reader should probably dictate "Credentials successfully configured for <CodeHostType>". On successfully removing credentials, the screen reader should probably dictate "Credentials successfully removed for <CodeHostType>".

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
@courier-new I think this is going to be a common issue and I've seen a good solution for this before. I've created an issue to focus on global announcements [here](https://github.com/sourcegraph/sourcegraph/issues/34504), will update with a WIP PR soon

**Comment by umpox** [comment]
@courier-new PR here, would be interested to get your 👀  https://github.com/sourcegraph/sourcegraph/pull/34506

**Comment by courier-new** [comment]
Punting on this one for now, see https://sourcegraph.slack.com/archives/C038K3PR2M7/p1651775064881929.

**Comment by courier-new** [comment]
You're right, we have the error covered with an `<Alert />`. I think we can close this one.
